cppframework/Foo.cpp

```diff
@@ -2,9 +2,6 @@
 #include <string>
 #include <iostream>
 
-#define EXPORT __attribute__((visibility("default")))
```

Comment:
I discovered two things. -fvisibility do not apply visibility to ObjC classes that are not in ObjC++ file. Changing ENABLE_TESTABILITY to NO in pure Xcode project makes symbols hide and tests no longer compile. For some reason though toggling ENABLE_TESTABILITY is not enough even after removal of -fvisibility flags...